src/Symfony/Component/DependencyInjection/Exception/ContextElementNotFoundException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Exception;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ContextElementNotFoundException extends InvalidArgumentException
```

Comment:
Again, I've reflected what is done in the `ParameterNotFoundException` ^^

The argument format is valid, but the argument is not valid in this context. Also that probably means it should lead to a change in your code, so it must be a `LogicException` (which `\InvalidArgumentException` extends). The behavior won't change at runtime.